packages/hh/src/index.ts

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+import { spawnSync } from "child_process";
+
+export async function main() {
+  let pathToHardhat;
+  try {
+    pathToHardhat = require.resolve("hardhat/internal/cli/cli.js", {
+      paths: [process.cwd()],
+    });
+  } catch (e) {
+    if (e.code === "MODULE_NOT_FOUND") {
+      console.error(
+        "You are not inside a Hardhat project, or Hardhat is not locally installed"
+      );
+    } else {
+      console.error(`[hh] Unexpected error: ${e.message}`);
+    }
+    process.exit(1);
+  }
+
+  const { status } = spawnSync(
+    "node",
+    [pathToHardhat, ...process.argv.slice(2)],
+    {
+      stdio: "inherit",
+    }
+  );
+
+  process.exitCode = status ?? 0;
```

Comment:
@alcuadrado according to `spawnSync` docs, the status part of the result is:

> The exit code of the subprocess, or null if the subprocess terminated due to a signal.

I don't really know how to handle the `null` case. Maybe it should be 1 if it was interrupted?